Runner/plans/meta-ar-ci-premerge.yaml

```diff
@@ -0,0 +1,21 @@
+metadata:
+    format: Lava-Test Test Definition 1.0
+    name: SmokeSanity
+    description: "Pre-merge LAVA plan to run AudioRecord and AudioPlayback on every PR"
+    maintainer:
+        - abbajaj@qti.qualcomm.com
+    os:
+        - openembedded
+    scope:
+        - functional
+    devices:
+        - rb3gen2
+
+run:
+    steps:
+        - cd Runner
+        - $PWD/suites/Multimedia/Audio/AudioPlayback/run.sh
```

Comment:
you should add `|| true` to all calls. Otherwise LAVA will exit if the script returns non zero exit code and the remaining scripts won't be excuted.